### DISCUSSION TITLE
Change table sort icon?

### DISCUSSION THREAD

**Discussion by PiousVenom** [rsuite/rsuite#2321] [Q&A]
Is it possible to change the sort icon for the tables?

**Comment by simonguo** [comment]
### rsuite v5 
The currently sorted icons cannot be changed. We will add a `renderSortIcon` prop on `<HeaderCell>`.

```tsx
const renderSortIcon = (type) => {
  if (type === "asc") {
    return <SortAscIcon />;
  } else if (type === "desc") {
    return <SortDescIcon />;
  }

  return <SortIcon />;
};

const CustomHeaderCell = React.forwardRef((props, ref) => (
  <HeaderCell {...props} ref={ref} renderSortIcon={renderSortIcon} />
));

const App = () => {
  return (
    ...
    <Column  sortable>
      <CustomHeaderCell>Id</CustomHeaderCell>
      <Cell dataKey="id" />
    </Column>
    ...
  );
};
```

> This feature will be released soon

### rsuite v4

In rsuite v4 it is possible to override font icons via css hack.
https://v4.rsuitejs.com/components/table/#Advanced
<img width="996" alt="image" src="https://user-images.githubusercontent.com/1203827/164141420-d9144bc8-39d6-47b5-befc-86c68a9138de.png">

**Comment by SevenOutman** [comment]
It's now available in 5.9.0
